cpp/react-native-hashing.cpp

```diff
@@ -23,8 +24,26 @@ namespace RNHashing {
             return jsi::Value(rt, jsi::String::createFromUtf8(rt, converter(input)));
         }
                         );
+        auto md5 = jsi::Function::createFromHostFunction(rt,
+                                                         jsi::PropNameID::forAscii(rt, "md5"),
+                                                         1,
+                                                         [](jsi::Runtime &rt, const jsi::Value &thisValue, const jsi::Value *args, size_t count) -> jsi::Value {
+            if (count == 0 || count > 1) {
+                jsi::detail::throwJSError(rt, "md5 requires one arguement");
+                return {};
```

Comment:
Returning `{}` from a function invokes default constructor of class you are returning from that function.

in case of `Value` it will invoke a constructor of undefined value.
https://github.com/facebook/react-native/blob/main/ReactCommon/jsi/jsi/jsi.h#L951

So this function returns undefined at also throws `throwJSError` error into runtime when you pass wrong arguments.

Will `throwJSError` be handled on JS level or will crash the app?

Reply:
As far as I know, It should be handled in js level in catch block.

Reply:
Should the developer send the wrong argument, this will create a js level error so it can be handled in a try-catch block in js. However i don't believe such a thing is required as the type definitions for the library will warn the user of the incorrect argument before we reach such a point.

Reply:
<img width="588" alt="Screen Shot 1400-11-27 at 21 18 06" src="https://user-images.githubusercontent.com/40071952/154325103-09ffb42c-2f89-4a2b-a575-9e23a98838ab.png">